Lib/test/test_netrc.py

```diff
@@ -138,7 +138,7 @@ def test_file_not_found_in_home(self):
     def test_file_not_found_explicit(self):
         self.assertRaises(FileNotFoundError, netrc.netrc,
                           file='unlikely_netrc')
-
+    @unittest.skipIf('vxworks' in sys.platform, 'VxWorks does not require or have a HOME directory')
     def test_home_not_set(self):
```

Comment:
Sprinkling verbose tests like this throughout the code is not the way. Instead, you should define a global flag at the top of the module, named based on the feature that is not on VxWorks.  E.g.
```
_HAVE_HOME = 'vxworks' not in sys.platform
```
Then, for the test functions:
```
@unittest.skipIf(not _HAVE_HOME)
```
Better yet perhaps is to add those flags to a central module, e.g. `platform`.  Doing it this way makes it more clear what is going on.  Also, if a new platform is added, the tests for platform specific features becomes easier to understand.